src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -1259,5 +1259,25 @@ private void OnWindowWheelEvent(object sender, Window.WheelEventArgs e)
             }
         }
 
+        /// <summary>
+        /// TouchArea can reset the view's touchable area.
+        /// If you set the TouchArea on an view, when you touch the view, the touch area is used rather than the size of the view
```

Comment:
Please add a period at the end of the sentence.
```suggestion
        /// If you set the TouchArea on an view, when you touch the view, the touch area is used rather than the size of the view.
```